charms/istio-pilot/config.yaml

```diff
@@ -3,6 +3,18 @@ options:
     type: string
     default: istio-gateway
     description: Name to use as a default gateway
+  image-configuration:
+    default: |
+      pilot-image: 'pilot'  # values.pilot.image
+      global-tag: '1.17.3'  # values.global.tag
+      global-hub: 'docker.io/istio' # values.global.hub
+      global-proxy-image: 'proxyv2' # values.global.proxy.image
+      global-proxy-init-image: 'proxyv2' # values.global.proxy_init.image
+      grpc-bootstrap-init: 'busybox:1.28'
```

Comment:
Where do those values come from? I can see the referenced doc but it is for another version and has different values at some places. This brings us to the next question as to how could one update those values in the case of bumping istio version for a new CKF release? Maybe, we will need to update more than just `global-tag`
Maybe we can proceed as it is now, but we 'll definitely need to tackle this soon.